app/src/main/res/color/primary_state_button_text_color.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
I feel this change was in response to the dark theme feedback. I am not entirely sure since we can't really test its usage since it's not being used. However, if this is to adjust for dark theme, we actually do this quite differently without selectors and there's `nightly/colors.xml`

Reply:
This is being used in another PR. I just want to get this landed.

Reply:
To be clear, there is additional work needed to add dark themes for "Jump back in" as well. This doesn't have to happen in this PR.

Reply:
I guess this is what makes the PR and review a bit weird since it's just xml.

Reply:
I believe I made it clear why this was broken out. You can see the link to the main PR in the description

Reply:
I am wondering what the additional work needed for the dark theme header since we already use `?primaryText` in `Header20TextStyle`. I think that's probably even more reasons we should just not have 2 headers and have to maintain both.

Reply:
Yes, and this will happen in another PR. Not relevant for this.